TrueCraft.Core/World/World.cs

```diff
@@ -93,8 +95,11 @@ public static World LoadWorld(string baseDirectory)
                 if (file.RootTag.Contains("Name"))
                     world.Name = file.RootTag["Name"].StringValue;
                 world.ChunkProvider = provider;
+
+                return world;
             }
-            return world;
+
+            throw new FileNotFoundException();
```

Comment:
If there is no world saved on the server then it won't generate one unless it catches an exception after first trying to load it. https://github.com/SirCmpwn/TrueCraft/blob/master/TrueCraft/Program.cs#L47